cypress/integration/grid_search.js

```diff
@@ -122,7 +122,7 @@ context("Grid Search", () => {
 		cy.get("@table")
 			.find('.grid-heading-row .search input[data-fieldtype="Duration"]')
 			.type("d");
-		cy.get("@table").find(".grid-body .rows .grid-row").should("have.length", 5);
+		cy.get("@table").find(".grid-body .rows .grid-row").should("have.length", 3);
```

Comment:
This is definitively correct.
Breaking down 7 days into one week which wouldn’t happen before, means there are less date strings with days…
This may change back once we opt for a default collecting only collecting days, hours, minutes and seconds.